src/bbmustache.erl

```diff
@@ -289,6 +289,10 @@ compile_impl([{'#', Keys, Tags, Source} | T], Data, Result, State) ->
                                              Result, Value), State);
       _ when Value =:= false ->
             compile_impl(T, Data, Result, State);
+      _ when Value =:= nil ->
+            compile_impl(T, Data, Result, State);
+      _ when Value =:= "" ->
```

Comment:
This line is never executed.
When the value is an empty string (= empty list), it handles at L287.

Elixier's string is Erlang binaries. So ...

```suggestion
      _ when Value =:= <<"">> ->
```